src/app/components/navbar/navbar.component.ts

```diff
@@ -152,11 +152,13 @@ export class NavbarComponent implements OnInit, OnDestroy {
     }
 
     private updateExtraTabs() {
-        if (this.experimentId != null) {
-            // only show experiment navigatio tabs if an experiment is active
-            this.extraTabs = this.experimentExtraTabs;
-        } else {
-            this.extraTabs = this.generalExtraTabs;
-        }
+        this.experimentId.subscribe(experimentId => {
```

Comment:
as the template now has different ngFor loops for the two extra tab groups this method is no longer needed. Just directly use the specific tab groups in the html template and remove `extraTabs` entirely.